Documentation/DailyCongressionalRecordEndpoint.md

```diff
@@ -58,57 +60,52 @@ Parent container for congressional record issues. A `<dailyCongressionalRecord>`
       - The daily Congressional Record's issue number.
    - `<volumeNumber>`
       - The daily Congressional Record's volume number.
-   - `<issueNumber>`
-      - The daily Congressional Record's issue number. 
+   - `<issueDate>`
+      - The date that the daily Congressional Record was issued.
    - `<congress>`
       - The Congress associated with the daily Congressional Record issue.
-   - `<SessionNumber>`
+   - `<sessionNumber>`
       - The session number. Possible values are "1" and "2". 
-   - `<URL>` 
-      - The URL to the entire issue of the daily Congressional Record. 
-    - `<UpdateDate>` 
-       - The date that the daily Congressional Record was updated.
-
-`<fullIssue>`
-
-  Container for full issue and sections. A `<fullIssue>` element may include the following children:
-
-  - `<entireIssue>`
-      - Container for entire issue items. An `<entireIssue>` element may include the following children:
-  - `<item>`
-      - Container for an entire issue item. An `<item>` element may include the following children:
-      - `<part>`
-         - The part of the daily Congressional Record issue.
-      -`<type>`
-         - The type of document that the daily Congressional Record is (e.g., PDF).
-      - `<URL> `
-         - The daily Congressional Record's URL.
-  - `<item>`
-      - Container for an entire issue item. An `<item>` element may include the following children:
-      - `<name>`
-         - The section name of the daily Congressional Record issue.
-      -`<startPage>`
-         - The start page of the daily Congressional Record section. 
-      - `<endPage>`
-         - The end page of the daily Congressional Record section. 
-- `<text>`
-      - Container for section text. A `<text>` element may include the following children:
-  - `<item>`
-      - Container for an entire issue item. An `<item>` element may include the following children:
-      - `<part>`
-         - The part of the daily Congressional Record issue.
```

Comment:
I haven't seen this prop return from the API. Is this supposed to be in the response?